Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentStart/WindowOpenHelper.js

```diff
@@ -0,0 +1,62 @@
+/* vim: set ts=2 sts=2 sw=2 et tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
```

Comment:
Do need these to be a IIFE? Didn't we decide that recently?

Reply:
Ah, that's right... Webpack wraps all of these anyway.